app/models/chargeback/consumption_history.rb

```diff
@@ -14,15 +14,18 @@ def self.for_report(cb_class, options, region)
 
         next unless options.include_metrics?
 
-        records = base_rollup.where(:timestamp => query_start_time...query_end_time, :capture_interval_name => 'hourly')
+        records = MetricRollup.where(:timestamp => query_start_time...query_end_time, :capture_interval_name => 'hourly')
         records = cb_class.where_clause(records, options, region)
-        records = Metric::Helper.remove_duplicate_timestamps(records)
```

Comment:
It seems like this will result in a change in behavior, maybe it's ok but I want to acknowledge it here - the original code in `remove_duplicate_timestamps` merges all metric rows it finds with the same timestamp - https://github.com/manageiq/manageiq/blob/b6bd309650ee46fedcdf6d9e00af4f06685bd7f8/app/models/metric/helper.rb#L139. This code will keep one of them and discard the other.